src/components/markdown-renderer/replace-components/highlighted-fence/highlighted-code/highlighted-code.tsx

```diff
@@ -42,18 +43,22 @@ export const HighlightedCode: React.FC<HighlightedCodeProps> = ({ code, language
   }, [code, language])
 
   return (
-    <code className={`hljs ${startLineNumber !== undefined ? 'showGutter' : ''} ${wrapLines ? 'wrapLines' : ''}`}>
-      {
-        highlightedCode
-          .map((line, index) => {
-            return <Fragment key={index}>
-              <span className={'linenumber'} data-line-number={(startLineNumber || 1) + index}/>
-              <div className={'codeline'}>
-                {line}
-              </div>
-            </Fragment>
-          })
-      }
-
-    </code>)
+    <Fragment>
+      <code className={`hljs ${startLineNumber !== undefined ? 'showGutter' : ''} ${wrapLines ? 'wrapLines' : ''}`}>
+        {
+          highlightedCode
+            .map((line, index) => {
+              return <Fragment key={index}>
+                <span className={'linenumber'} data-line-number={(startLineNumber || 1) + index}/>
+                <div className={'codeline'}>
+                  {line}
+                </div>
+              </Fragment>
+            })
+        }
+      </code>
+      <div className={'text-right'} style={{ marginTop: '-31px' }}>
```

Comment:
static style

Reply:
Yeah, I'll put that into an `.scss` file

Reply:
done. The class lies in a separate file for a future PR